Makefile

```diff
@@ -69,3 +69,10 @@ coveralls:
 BENCH ?= .
 bench:
 	@$(foreach pkg,$(PKGS),go test -bench=$(BENCH) -run="^$$" $(BENCH_FLAGS) $(pkg);)
+
+.PHONY: updatebench
+updatebench:
+	@if ! bash --version | grep 'version 4' > /dev/null; then \
+		echo "You need to use bash 4 to run update_benchmarks.sh"; \
```

Comment:
Aw man :) Ya I mean I could do the whole thing in golang, I was just messing around. I can make it not-bash-4 too (bash 3 doesn't do associative arrays). Could I do that, and then later on we make this golang?